src/application/components/ExternalLink.tsx

```diff
@@ -0,0 +1,8 @@
+import { type ComponentProps, forwardRef } from "react";
+
+export const ExternalLink = forwardRef<
+  HTMLAnchorElement,
+  ComponentProps<"a"> & { href: string }
+>((props, ref) => {
+  return <a {...props} ref={ref} />;
```

Comment:
Since we are moving to this, let's add `target="_blank" rel="noopener noreferrer"` as props here as well since external links on the web side don't open unless they are in a new tab.